packages/compass-e2e-tests/helpers/commands/workspace-tabs.ts

```diff
@@ -14,31 +16,36 @@ async function closeTab(
   selectorOptions: WorkspaceTabSelectorOptions,
   autoConfirmTabClose: boolean
 ): Promise<void> {
-  await browser.clickVisible(
-    browser
-      .$(Selectors.workspaceTab(selectorOptions))
-      .$(Selectors.CloseWorkspaceTab)
-  );
-
   // wait until the tab goes away and if the confirmation modal opens, maybe confirm
-  await browser.waitUntil(async () => {
-    if (autoConfirmTabClose) {
-      // Tabs in "dirty" state can't be closed without confirmation
-      if (await browser.$(Selectors.ConfirmTabCloseModal).isExisting()) {
-        await browser.clickVisible(
-          browser.$(Selectors.ConfirmTabCloseModal).$('button=Close tab')
-        );
-        await browser
-          .$(Selectors.ConfirmTabCloseModal)
-          .waitForDisplayed({ reverse: true });
+  await browser.waitUntil(
+    async () => {
+      // keep retrying the click :(
```

Comment:
Drive-by. This was something I moved out recently because clicking again won't work if the modal obscures it anyway, but it seems like it was necessary because it became flaky again.